framework/js/components/overlaySrcFocus.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+class OverlaySrcFocus {
+
+	constructor() {
+		// Aouter un event listener sur les boutons qui ouvrent un overlay / modale
+		if (document.querySelector('*[data-open-overlay="true"]') !== null) {
+			document.querySelectorAll('*[data-open-overlay="true"]').forEach((element) => {
+                element.addEventListener('click', () => {
+                    this.designElementAsOverlaySrc(element);
+                })
+            });
+		}
+
+		document.addEventListener('overlay:hide', () => { this.removeDesignatedOverlaySrc() });
```

Comment:
Préférer : document.addEventListener('overlay:hide', this.removeDesignatedOverlaySrc.bind(this));